sandbox.sh

```diff
@@ -173,7 +173,7 @@ start_deku_cluster() {
   SERVERS=()
   echo "Starting nodes."
   for i in ${VALIDATORS[@]}; do
-    deku-node "$data_directory/$i" --listen-prometheus=9000 &
+    deku-node "$data_directory/$i" --listen-prometheus=900$i &
```

Comment:
We might want to revert this before merge, but that is something @Zett98 and @aguillon should decide on.  And maybe @d4hines should have a word on it.

Reply:
I'm ok with this change.

Reply:
I'm ok with this change but we should probably move it to a proper configuration file one day.